ddtrace/debugging/_signal/utils.py

```diff
@@ -136,6 +142,14 @@ def capture_exc_info(exc_info):
     }
 
 
+def redacted_value(v: Any) -> dict:
+    return {"type": qualname(type(v)), "notCapturedReason": "redactedIdent"}
```

Comment:
need to align with java/dotnet implementation. their value is "redacted" for both cases.

Reply:
That was recently changed in the RFC